debian/control

```diff
@@ -0,0 +1,19 @@
+Source: souffle
+Maintainer: Nathan Keynes <nathan.keynes@oracle.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: autoconf, automake, bison, debhelper (>= 9), flex, libsqlite3-dev, openjdk-7-jdk | openjdk-8-jdk
+Homepage: https://github.com/oracle/souffle
+Vcs-Git: https://github.com/oracle/souffle.git
+
+Package: souffle
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, g++, jarwrapper, openjdk-7-jre | openjdk-8-jre
```

Comment:
As I understand it you can build multiple packages from one source package (e.g.https://packages.ubuntu.com/source/trusty/samba) rather than splitting it up.

But sure, we can do this later.